### ISSUE TITLE
Please release 1.4.1

### ISSUE THREAD

**Issue by timmolter** [mmazi/rescu#17]
Hi Matija,

I'd like to release XChange 1.10.0.

One quick question though...

I've received an issue regarding an exception sneaking though here: https://github.com/timmolter/XChange/issues/192

I've verified this:

```
com.fasterxml.jackson.core.JsonParseException "Illegal character ((CTRL-CHAR, code 31)): only regular white space (\r, \n, \t) is allowed between tokens
```

Do you think ResCU could and should catch and wrap this as well?

~Tim

**Comment by timmolter** [comment]
More info on the XChange issue #192...

Yes, a 500 code is returned, but I guess jackson still tries to parse the response. From a bit of Googling, the illegal character stuff might be because the response is GZipped!?!

![screen shot 2013-11-09 at 1 58 48 pm](https://f.cloud.github.com/assets/738972/1506680/e9e0e36c-493e-11e3-90ff-9ffefd30887e.png)

**Comment by timmolter** [comment]
`WithdrawalFundsDemo` is the class in question if you'd like to run it yourself.

**Comment by mmazi** [comment]
I fixed the problem in rescu:
- Added support for gzipped responses,
- Recover when parsing of response body in case of a non-200 response code fails to produce a valid exception object from the json source.

**Comment by mmazi** [comment]
Released rescu as 1.5.0 (there were some API changes concerning exceptions, hence the minor version increase).
